src/components/DashboardHome/Sidebar.jsx

```diff
@@ -1,11 +1,37 @@
 import React from 'react';
+import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import Links from '../Links/Links';
 import linksData from './sampleLinks';
+import SidebarBlock from './SidebarBlock';
 
 const Sidebar = () => (
   <>
-    <Links id={linksData.id} title={linksData.title} links={linksData.links} />
+    <SidebarBlock title="Program Documents" className=".mb-4">
```

Comment:
`className` prop should just be `mb-4` (without the `.`). The `.` is included when you're referencing the class name in CSS. Same for the 2nd `SidebarBlock` component.